### DISCUSSION TITLE
DoubleCmd is indeed a great tool!

### DISCUSSION THREAD

**Discussion by vagra** [doublecmd/doublecmd#1547] [General]
For years, I had been using Q-Dir until its features became insufficient for my needs. Then I switched to Directory Opus for a few years. It could indeed meet my daily requirements, especially with the ability to lock tabs and bookmark frequently used paths. However, it requires payment and is not open-source, plus the interface is not very appealing.

In the past few days, I tried out DoubleCmd, and it's really nice! It can also lock tabs and bookmark commonly used directories. Its features are almost on par with Directory Opus, but DoubleCmd is open-source, free, and has a more harmonious and visually pleasing interface. I have now uninstalled Directory Opus and will be using DoubleCmd going forward.

**Comment by ags1234** [comment]
Out of curiosity, what feature did you use in Directory Opus that you cannot find in DoubleCmd?

**Reply by vagra** [reply]
There are several features in Directory Opus that I find useful and that DoubleCmd either lacks or is not yet fully developed:

### 1
In Directory Opus, when selecting files in the file list, it's possible to choose only the file name column, whereas in DoubleCmd, the selection encompasses a whole column including extension, size, modification time, and attributes. However, in DoubleCmd, to navigate to the parent folder with a double-click or to access the right-click menu, you need to click in the unselected space, which is relatively small.

### 2
The tabs in Directory Opus can not only be arranged above or below the file list but also to the left and right. When there are many locked tabs, having them arranged on the left or right side is very convenient!

### 3
Directory Opus features a theme system with a wide selection of themes to choose from, whereas DoubleCmd only offers two themes: light and dark.

### 4
Directory Opus runs more stably, whereas DoubleCmd sometimes experiences lagging or crashes.

**Reply by vagra** [reply]
As shown in this screenshot.
![20240401_090309](https://github.com/doublecmd/doublecmd/assets/1576687/b092b613-c066-46ae-a393-742866e4b9a4)

**Comment by Augusto7743** [comment]
Double Commander is my file manager utility being the top list of software for Linux.
Strangely several others file manager has few options mainly as show paths and directories.
Double Commander creator was smart doing it.

**Comment by ags1234** [comment]
From your picture and others on the net, Directory Opus looks very good. I wonder if it can be easily controlled by keyboard.

The nice thing about file managers like Total Commander, Double Commander, Double Commander, Free Commander, Multi Commander, EF Commander, Far Manager, Dos Navigator is that they can easily be controlled by keyboard, and one can become very productive after getting used to this workflow.

**Reply by vagra** [reply]
To put it simply, Total Commander operates similarly to VIM, with its own set of keyboard shortcuts. On the other hand, Directory Opus is distinct in that it doesn't belong to the same system as Total Commander. Instead, it aligns more closely with the typical operating habits of Windows itself, making it better suited for users accustomed to Windows. DoubleCmd, on the other hand, clearly falls within the Total Commander system. After using it for a few days, I've noticed some differences in operating habits compared to Windows.

**Reply by ags1234** [reply]
Windows Explorer's way of managing files is far less efficient than that of orthodox file managers, like DC. If you learn to control it with one hand on the keyboard and the other on the mouse. Also, try to be aware of key combinations.
The mouse has to be positioned on the right spot on the screen and that takes time.
The keys are always in the same place :)

**Comment by vagra** [comment]
The two commonly used functions of searching and batch renaming in DoubleCmd are much better than Directory Opus, whether in terms of interface aesthetics, consistency, or functionality completeness.

**Comment by ags1234** [comment]
There is also a compare differences between two files function.
And a synchronize directories function, but his one is missing something: it does not search for empty directories as far as I know.

**Comment by mateus3c** [comment]
I was wondering what's the feature difference between DoubleCommander, MultiCommander and TotalCommander

**Comment by ags1234** [comment]
Just a few differences that I can think of now:

DC has problems with elevation in Windows. I had to access files from another user's account, on another SSD and I had to use TC, because DC did not let me, not knowing how to ask for elevation.
DC cannot move files to zip archives.
Synchronize directories in DC cannot search for empty directories. So when you synchronize and DC tells you the parts are identical, they may not be.

I don't know MultiCommander that well, but I admire the way it's files are organized on disk, it's neat and tidy, the small size on disk, The way it's settings appear in a files panel is unconventional and to my knowledge, unique to MC.
It cannot compress files to RAR archive. The author was not too open to suggestions.
